Jellyfin/Jellyfin.php

```diff
@@ -43,7 +43,7 @@ private function getAttrs()
     {
         return [
             "headers" => [
-                "X-MediaBrowser-Token" => $this->config->password,
+                "Authorization" => "MediaBrowser Token=\"" . urlencode($this->config->password) . "\"",
```

Comment:
Thanks for this. :smile: I would like to suggest that we add a client "Heimdall", possibly with a version number.

Reply:
What's the best way to fetch the version number?

Reply:
@imnotjames not sure if the version gets exposed to the apps. There is a Setting model that is used to interact with the DB but I'm unsure if the settings a globally available to an app.

I would also be fine with the name. Just seemed better to at least have a client name. I did not get the impression that the version was mandatory.

Reply:
Yeah, I can't find anything easily to get it reliably.  Marking as resolved, reopen if any issues.